en/asgardeo/docs/guides/api-security/aws/securing-aws-http-apis.md

```diff
@@ -0,0 +1,279 @@
+---
+template: templates/quick-start.html
+heading: Secure a Pharmacy User API with AWS API Gateway and Asgardeo
+description: This guide walks you through securing a Pharmacy User API using AWS API Gateway and WSO2 Asgardeo. You'll deploy a Lambda function, expose it via an HTTP API, and protect the endpoint using JWT validation powered by Asgardeo.
+what_you_will_learn:
+  - Deploy a serverless AWS Lambda endpoint
+  - Secure the API using AWS API Gateway's JWT authorizer
+  - Configure OAuth2 application and API scopes in Asgardeo
+  - Protect API access using JWTs and test secured endpoints
+prerequisites:
+  - About 20 minutes
+  - AWS Account with CLI access
+  - <a href="https://asgardeo.io" target="_blank" rel="noopener noreferrer">Asgardeo account</a>
+  - Tools: <code>curl</code> and <code>jq</code>
+  - Familiarity with shell and JSON
+---
+
+## Step 1: Create and Deploy the Lambda Function
+
+### 1. Create IAM Role for Lambda Execution
+```bash
```

Comment:
Shall we add a description?

Reply:
Addressed in [e4fc8aa](https://github.com/wso2/docs-is/pull/5253/commits/e4fc8aa8a7f3140b9f3447455ff995b2ad2ef2ac)